sycl/doc/extensions/SPIRV/SPV_INTEL_vector_compute.asciidoc

```diff
@@ -149,14 +156,28 @@ Modify Section 3.31, Capability, add the following rows the 'Capability' table:
 |====
   2+^| Capability      |     Implicitly Declares    | Enabled by Extension
 | 5617 | *VectorComputeINTEL* +
-Enables the use of *Private* storage class, *SharedLocalMemorySizeINTEL* execution mode and *VectorComputeFunctionINTEL*, *VectorComputeVariableINTEL*, *StackCallINTEL*, *SIMTCallINTEL*, *FuncParamIOKindINTEL* and *GlobalVariableOffsetINTEL* decorations.
+Enables the use of *Private* storage class, *SharedLocalMemorySizeINTEL* execution mode and *VectorComputeFunctionINTEL*, *VectorComputeVariableINTEL*, *StackCallINTEL*, *SIMTCallINTEL*, *FuncParamIOKindINTEL*, *GlobalVariableOffsetINTEL*
+*SingleElementVectorINTEL* decorations and *OpTypeBufferSurfaceINTEL* type.
 |*VectorAnyINTEL* | *SPV_INTEL_vector_compute*
 | 5619 | *VectorAnyINTEL* +
 Uses OpTypeVector to declare vectors with any number of components.
 || *SPV_INTEL_vector_compute*
 |====
 --
 
+Modify Section 3.36.6. Type-Declaration Instructions,  add the end of type instructions list:
+--
+[cols="1,1,1",width="60%"]
+|=====
+2+|*OpTypeBufferSurfaceINTEL* +
```

Comment:
Why can't we re-use existing [`OpTypeOpaque`](https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#OpTypeOpaque) for that purpose?

Reply:
**OpTypeBufferSurfaceINTEL** will be extended in the next update to have [Access Qualifier](https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#Access_Qualifier) operand that cannot be applied to **OpTypeOpaque**.